tests/unit/test_influxbackend.py

```diff
@@ -1,83 +1,523 @@
-#        value = '1324'
-#        timestamp = now()
-#
-#        with self.assertRaises(InvalidNamespaceError):
-#            self.influxbackend.save(namespace, value, timestamp)
-#
-#    def test_save_failed_namespace_not_string(self):
-#        """Test case where namespace is not a string type."""
-#        namespace = 1234
-#        value = '1324'
-#        timestamp = now()
-#
-#        with self.assertRaises(TypeError):
-#            self.influxbackend.save(namespace, value, timestamp)
-#
-#    def test_save_failed_invalid_value(self):
-#        """Test case where the value to store is not float."""
-#       namespace = 'kytos.kronos.telemetry.switches.1.interfaces.232.bytes_in'
-#        value = 'invalid value'
-#        timestamp = now()
-#
-#        with self.assertRaises(ValueConvertError):
-#            self.influxbackend.save(namespace, value, timestamp)
-#
-#    def test_get_success(self):
-#        """Test to check success in data retrieving in Influx Backend."""
-#       namespace = 'kytos.kronos.telemetry.switches.1.interfaces.232.bytes_in'
-#        value = 1324
-#        timestamp = now()
-#
-#        self.influxbackend.save(namespace, str(value), timestamp)
-#
-#        results = self.influxbackend.get(namespace, start=timestamp)
-#        expected_store_value = [[timestamp, value]]
-#
-#        self.assertEqual(expected_store_value, results)
-#
-#    def test_delete_success(self):
-#        """Test to check success deleting data in Influx Backend."""
-#       namespace = 'kytos.kronos.telemetry.switches.1.interfaces.232.bytes_in'
-#        value = 1324
-#        timestamp = now()
-#
-#        with self.assertRaises(NamespaceNotExistsError):
-#            self.influxbackend.save(namespace, value, timestamp)
-#            self.influxbackend.delete(namespace, str(value), timestamp)
-#            self.influxbackend.get(namespace, start=timestamp)
+"""Module to test the Influx Backend.
+
+Unit tests to check fail and success cases in Influx Backend module. The tests
+can be organized in two parts. The first consists the tests to private methods
+for the module are the tests for the InfluxBackend class.
+isort:skip_file
+"""
+import sys
+
+# pylint: disable=wrong-import-order,wrong-import-position
+
+from unittest import TestCase, mock
+
+sys.modules['influxdb'] = mock.MagicMock()
+sys.modules['influxdb.exceptions'] = mock.MagicMock()
+
+import napps.kytos.kronos.backends.influx as influx_module
+from napps.kytos.kronos.utils import (BackendError, NamespaceError,
+                                      ValueConvertError)
+
+# pylint: enable=wrong-import-order,wrong-import-position
+# pylint: disable=R0904,E0012, Too many public methods
+
+
+class TestInfluxBackend(TestCase):
+    """Test methods in Influx Backend."""
+
+    def setUp(self):
+        """Start Influx Backend."""
+        settings = mock.MagicMock()
+        self.influxbackend = influx_module.InfluxBackend(settings)
```

Comment:
Suggestion: the module is already named `test_influxbackend`, to `self.backend` is enough to tell which one we're using here.

The same goes to `influx_module`: `influx` should do it.

Reply:
Good point!
I'll apply it. Thanks.